client/src/components/common/applicant-info-2.vue

```diff
@@ -155,7 +213,8 @@ import NameRequestMixin from '@/components/mixins/name-request-mixin'
 @Component({
   components: {
     ApplicantInfoNav
-  }
+  },
+  directives: { mask }
```

Comment:
Is `Vue-the-mask` used for validating formats? If so, a good way to keep Regex out of the rules!

Reply:
to validate phone number